controllers/errors/custom-error-controllers.js

```diff
@@ -5,5 +5,10 @@ exports.customErr = (err, req, res, next) => {
     res.status(400).send({ msg: "invalid sort query" });
   } else if (err.status === 400 && err.msg === "invalid order query") {
     res.status(400).send({ msg: "invalid order query" });
+  } else if (
+    err.status === 404 &&
+    err.msg === "no articles found for this topic"
+  ) {
+    res.status(404).send({ msg: "no articles found for this topic" });
```

Comment:
Bear in mind that if the error object has a `status` and a `msg` then it's guaranteed to be one of your custom promise rejections - you'd then be free to use them directly rather than repeating here